package.json

```diff
@@ -22,6 +22,7 @@
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-progressive-image": "^0.6.0",
+    "react-responsive": "^8.1.0",
```

Comment:
No need for this, chakra-ui has the responsive utilities by default

Reply:
Removed react-responsive.